Gemfile

```
@@ -8,7 +8,7 @@ group :test do
   gem 'minitest'
   gem 'minitest-spec-context'
   gem 'mocha'
-  gem 'rake', '~> 10.1.0'
+  gem 'rake', '~> 13.0.6'
```

Please pin to x.y versions. Pinning to x.y.z means you'll need to update it to allow x.(y+1).z. Most projects stick to [semver](https://semver.org/) and that means 13.1 should still be compatible with 13.0.
```suggestion
  gem 'rake', '~> 13.0'
```

---

Hmm, this is interesting. Thanks for the mention. It's updated now.